OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentRegistrations/StudentRegistrationServiceTests.Validations.RetieveById.cs

```diff
@@ -18,10 +18,10 @@ public partial class StudentRegistrationServiceTests
         public async void ShouldThrowValidationExceptionOnRetrieveWhenIdIsInvalidAndLogItAsync()
         {
             // given
-            Guid randomStudentId = default;
-            Guid inputStudentId = randomStudentId;
-            Guid randomRegistrationId = default;
-            Guid inputRegistrationId = randomRegistrationId;
+            Guid invalidStudentId = Guid.Empty;
```

Comment:
If we are testing both Id, then we need to update the problem details. Please connect with me or Hassan to understand this.